config/candidates-api.yml

```diff
@@ -0,0 +1,111 @@
+---
+openapi: 3.0.0
+info:
+  version: v1
+  title: Apply candidates API
+  contact:
+    name: DfE
+    email: becomingateacher@digital.education.gov.uk
+  description: |
+    API for candidates from DfE’s Apply for teacher training service.
+servers:
+- description: Sandbox (test environment)
+  url: https://sandbox.apply-for-teacher-training.service.gov.uk/candidates-api
+- description: Production
+  url: https://www.apply-for-teacher-training.service.gov.uk/candidates-api
+paths:
+  "/candidates":
```

Comment:
I think we could just have the `updated_since` version and always insist on that date being present?